command/forceleave/forceleave.go

```diff
@@ -47,10 +52,14 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
-	err = client.Agent().ForceLeave(nodes[0])
+	if c.prune {
+		err = client.Agent().ForceLeavePrune(nodes[0])
+	} else {
+		err = client.Agent().ForceLeave(nodes[0])
+	}
+
 	if err != nil {
 		c.UI.Error(fmt.Sprintf("Error force leaving: %s", err))
-		return 1
```

Comment:
If the leave failed we'd probably still want the CLI command to fail (i.e. exit non-zero).